dev-python/py-gfm/py-gfm-1.0.2.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Github-Flavored Markdown for Python-Markdown"
+HOMEPAGE="https://github.com/Zopieux/py-gfm"
+SRC_URI="https://github.com/Zopieux/py-gfm/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+RDEPEND="dev-python/markdown[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
```

Comment:
likely unnecessary, also no tests?

Reply:
This should be the test: https://github.com/Zopieux/py-gfm/blob/master/tests/test_gfm.py and there are lots of "some-code" "foo" ecc... I don't think test actually works or do something useful